examples/link/link.html

```diff
@@ -121,6 +128,7 @@
                   style={styles.urlInput}
                   type="text"
                   value={this.state.urlValue}
+                  onKeyUp={this.onLinkInputKeyDown}
```

Comment:
Let's stick with `onKeyDown` for consistency, since `onKeyUp` isn't used elsewhere in the framework at this point.